sound/soc/intel/boards/sof_realtek_common.h

```diff
@@ -39,4 +39,9 @@ void sof_rt1015_codec_conf(struct snd_soc_card *card);
 #define RT1308_DEV0_NAME	"i2c-10EC1308:00"
 void sof_rt1308_dai_link(struct snd_soc_dai_link *link);
 
+#define RT1019_CODEC_DAI	"HiFi"
+#define RT1019_DEV0_NAME	"RTL1019:00"
```

Comment:
I think the 1019 is the same as the 1015p, according to this:
````c
soc/codecs/rt1015p.c:   { "RTL1019", 0},
````

We already have support for the 1015p in sof_realtek_common, so please double-check with @brentlu that this does not interfere or duplicate the existing code.

Reply:
Thanks Peirre for reviewing the changes.
Brent is ok with the changes. https://github.com/thesofproject/linux/pull/3614#pullrequestreview-953030792